x-pack/plugin/index-lifecycle/src/main/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleService.java

```diff
@@ -61,6 +61,10 @@ public IndexLifecycleService(Settings settings, Client client, ClusterService cl
         clusterService.addListener(this);
     }
 
+    public PolicyStepsRegistry getStepsRegistry() {
```

Comment:
Why do we need to expose the registry here? I would rather keep this internal

Reply:
no, it was just being used by the Task to validate, I can only expose that level of operation from the service itself